.github/workflows/build.yml

```diff
@@ -300,3 +300,14 @@ jobs:
         with:
           sa_key: ${{ secrets.GCP_CREDENTIALS }}
           previewctl_hash: ${{ needs.build-previewctl.outputs.previewctl_hash }}
+
+  deploy-webapp-staging:
+    name: "Deploy WebApp SaaS to staging"
+    needs: [ install, monitoring ]
+    runs-on: ubuntu-latest
+    if: |
+      (needs.configuration.outputs.is_main_branch != 'true')
+    steps:
+      - uses: gitpod-io/ops/.github/workflows/webapp-deploy-staging.yml@main
+        with:
+          version: ${{needs.configuration.outputs.version}}
```

Comment:
you'll have to pass the token somewhere here, I think. Added a hold so you can change it.